### ISSUE TITLE
[bug] Strange error message when lockfile cannot be found

### ISSUE THREAD

**Issue by KerstinKeller** [conan-io/conan#8918]
When I am passing a nonexisting file to conan install for a lockfile
```
conan install -l=./lockfiles/nonexisting.lock .
```
I get a strange error message:
```
ERROR: Missing lockfile in: /home/project/lockfiles/nonexisting.lock/conan.lock
``` 

Somehow the error message suggests, that the `-l` is not interpreted as a path to a lockfile, but as a folder (an implicitly appends a `conan.lock`.
And the documentation says that it expects a file:

```
  -l LOCKFILE, --lockfile LOCKFILE
                        Path to a lockfile
```

I would expect the error message would be:
```
ERROR: Missing lockfile in: /home/project/lockfiles/nonexisting.lock
```

I am using the latest Conan version (1.36.0)

**Comment by czoido** [comment]
Hi @KerstinKeller,
Thanks a lot for reporting, the message is in fact misleading and should be improved.

**Comment by memsharded** [comment]
This is a small UX issue, will be automatically fixed in Conan 2.0 (lockfile name will be mandatory), moving it to 2.0 milestone.

**Comment by memsharded** [comment]
Solved in 2.0-beta.1, feedback welcome
